acceptance/helper.rb

```diff
@@ -228,7 +228,14 @@ def puppetdb_pids(host)
     pids
   end
 
-  def start_puppetdb(host, test_url="/pdb/meta/v1/version")
+  def start_puppetdb(host, legacy=false)
```

Comment:
Its interesting, not sure how far we want to go down any road - this code is going soon once we release.